### PR TITLE
Refactor: rename EventTicketType props

### PR DESCRIPTION
**Pull request by pauloiankoski**
Depends on https://github.com/impress-org/givewp/pull/7233

## Description

This pull-request proposes some changes detected after starting developing the new Events feature:
- Assign the recently inserted ID to the model;
- Rename the `max_available` column as `capacity` to give it more clarity of what is it about;
- Rename the `label` column as `title` to align with Events;

## Affects

EventTicketType data

## Pre-review Checklist

-   [ ] Acceptance criteria satisfied and marked in related issue
-   [x] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [ ] Reviewed by the designer (if follows a design)
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

### CODE REVIEW

File: src/EventTickets/Migrations/CreateEventTicketTypesTable.php [hunk 1]
```diff
@@ -43,8 +44,9 @@ public function run() {
 			event_id INT UNSIGNED NOT NULL,
 			label TEXT NULL,
 			description TEXT NULL,
-			price INT UNSIGNED NULL,
-			max_available INT UNSIGNED NULL,
+			price DECIMAL(18,10) NOT NULL,
+			max_tickets_available INT UNSIGNED NULL,
```
Comment (kjohnson) [inline src/EventTickets/Migrations/CreateEventTicketTypesTable.php:48]: Why did you change the column name?

File: src/EventTickets/Models/EventTicketType.php [hunk 2]
```diff
@@ -131,7 +127,15 @@ public function event(): ModelQueryBuilder
...
     public function eventTickets(): ModelQueryBuilder
     {
-        return give(EventTicketRepository::class)->queryByEventId($this->eventId);
+      return give(EventTicketRepository::class)->queryByEventId($this->eventId);
+    }
+
+    /**
+     * @unreleased
+     */
+    public function countSoldTickets(): int
```
[the remark this answers is not in the document]
Comment (pauloiankoski) [reply src/EventTickets/Models/EventTicketType.php]: Resolved: 63409c07740ebf8ba8f450df12ca40efce5e7231

File: src/EventTickets/Migrations/CreateEventTicketTypesTable.php (hunk 1 above)
Comment (pauloiankoski) [reply src/EventTickets/Migrations/CreateEventTicketTypesTable.php]: Resolved: a5d894416d2d9e6ac8e594326ca2769773d3615d

**Review by kjohnson — COMMENTED** [review]
@pauloiankoski could you remove the changes that rename the `ticket_close_datetime`  property as status so that we can discuss that change separately? This PR is changing multiple things and the PR title is vague.

**Comment by pauloiankoski** [comment]
> @pauloiankoski could you remove the changes that rename the `ticket_close_datetime` property as status so that we can discuss that change separately? This PR is changing multiple things and the PR title is vague.

Reverted on c0c005f44b6f88caff4c9370beb2b580e9129d1d and 75160cbf050e3eea41bcc4036fe9d50c428d6517

File: src/EventTickets/Migrations/CreateEventTicketTypesTable.php [hunk 3]
```diff
@@ -42,10 +42,10 @@ public function run() {
         $sql = "CREATE TABLE $table (
 			id INT UNSIGNED NOT NULL AUTO_INCREMENT,
 			event_id INT UNSIGNED NOT NULL,
-			label TEXT NULL,
+			title TEXT NULL,
 			description TEXT NULL,
-			price INT UNSIGNED NULL,
-			max_available INT UNSIGNED NULL,
+			price DECIMAL(18,10) NOT NULL,
```
Comment (kjohnson) [inline src/EventTickets/Migrations/CreateEventTicketTypesTable.php:47]: This should be reverted to an unsigned integer.

Comment (kjohnson) [reply src/EventTickets/Migrations/CreateEventTicketTypesTable.php:47]: In general, currency values should be stored in minor amounts, which is a best practice. We diverge from this in some places, but that is largely for reasons of backwards compatibility.

Comment (pauloiankoski) [reply src/EventTickets/Migrations/CreateEventTicketTypesTable.php]: Reverted on 854403bc6894a4baae4d761d07c832a00080f698

**Review by kjohnson — APPROVED** [review]
